service/orchestrator/orchestrator.go

```diff
@@ -213,51 +213,56 @@ func (s *Service) StoreAssessmentResults(stream orchestrator.Orchestrator_StoreA
 
 	for {
 		result, err = stream.Recv()
-		if err == io.EOF {
-			log.Infof("Stopped receiving streamed assessment results")
-			return stream.SendAndClose(&emptypb.Empty{})
+
+		if err != nil {
+			// If no more input of the stream is available, return SendAndClose `error`
+			if err == io.EOF {
+				log.Infof("Stopped receiving streamed assessment results")
+				return stream.SendAndClose(&emptypb.Empty{})
+			}
+
+			return err
 		}
 
 		err = s.handleResult(result)
 		if err != nil {
-			return fmt.Errorf("error handle assessment result: %w", err)
+			return status.Errorf(codes.Internal, "Error while handling assessment result: %v", err)
```

Comment:
I think error messages should begin with lowercase. It is definitely the case for `fmt.Errorf`, not 100% sure about `status.Errorf`

Reply:
For me it's ok, if every error starts with lowercase.

Reply:
@oxisto but the log.Errorf start with uppercase?!?

Reply:
Lowercase. See https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
> @oxisto but the log.Errorf start with uppercase?!?

Yes. See https://github.com/golang/go/wiki/CodeReviewComments#error-strings. The idea is that the error message should be lowercase because it is often logged as "The error is: %v" in a log function (such as log.Errorf). Log messages can/should start with uppercase.